packages/mwp-core/src/renderers/server-render.jsx

```diff
@@ -97,23 +99,8 @@ const getMedia = (userAgent: string, userAgentDevice: string) => {
 
 /**
  * Using the current route information and Redux store, render the app to an
- * HTML string and server response code.
- *
- * There are three parts to the render:
- *
- * 1. `appMarkup`, which corresponds to the markup that will be rendered
- * on the client by React. This string is built before the full markup because
- * it sets the data needed by other parts of the DOM, such as `<head>`.
- * 2. `htmlMarkup`, which wraps `appMarkup` with the remaining DOM markup.
- * 3. `doctype`, which is just the doctype element that is a sibling of `<html>`
- *
- * @param {Object} renderProps
- * @param {ReduxStore} store the store containing the initial state of the app
- * @return {Object} the statusCode and result used by Hapi's response API
+ * HTML string and server response code, with optional cookies to write
  */
-type RenderResult =
```

Comment:
If this type is removed here, what is the return type being used here https://github.com/meetup/meetup-web-platform/pull/612/files#diff-d7329f1db146f21557dc646dcd0d2350R112

Reply:
this is defined in `flow-typed/platform.js` - looks like this is just an old re-declaration of the same type available globally